charts.py

```diff
@@ -0,0 +1,145 @@
+import os
+import json
+import numpy as np
+
+from flask import (
+    Flask, flash, render_template,
+    redirect, request, session, url_for)
+from flask_pymongo import PyMongo
+
+if os.path.exists('env.py'):
+    import env
+
+
+charts = Flask(__name__)
+charts.config['MONGO_DBNAME'] = os.environ.get('MONGO_DBNAME')
+charts.config['MONGO_URI'] = os.environ.get('MONGO_URI')
+charts.secret_key = os.environ.get('SECRET_KEY')
+
+mongo = PyMongo(charts)
+
+# collection
+data_set_coll = mongo.db.SampleData
```

Comment:
I would not load this globally, but in the view. Also, let's not use `SampleData`, but an actual collection name e.g. `nace_labels` or similar

Reply:
The SampleData is the name of my DB and correspond to the json file basically. I believe this will be changed when a DB will be set for the project globally as our own is not going to be used. 
The SampleData DB is to show that it is not a "functioning" DB for the global project.
I Let me know what you think but if I change it now, it will have to be change later as well anyway.